### ISSUE TITLE
[BUGS] List of gameplay bugs I encountered

### ISSUE THREAD

**Issue by Reziax** [xJon/Hexxit-II#227]
Hi, just a quick list of bugs that me and others ran into while playing:

1. "Dirt Scorpion Armor" set from Mo'Creatures gives health boost II but every couple seconds it resets the health to normal and then reapplies, which basicially turns it into a saturation drain. I assume this happens because it applies a few seconds of the buff periodically instead of once infinitely. 

2. Bibliocraft's armorstand fails to display the armor in it's inventory. Happens with fresh install without any shaders/optifine.

3. The enchantments from "fish's undead rising" appear as enchanted books in all sorts of containers/drops, but it's impossible to add them to equipment via the anvil in survival mode (works in creative). I assume the intention was to make them loot-exclusive and not acquirable through the ench+ table? But something went wrong and they can't be applied.

There's more but I lost some notes, so I will update this post when I remember lol

**Comment by xJon** [comment]
Thank you for the reports! The first issue will be fixed in the next modpack update with the new update of Mo' Creatures Extended, the second issue is on the mod's end, which unfortunately is no longer actively developed.
Regarding the third issue, are you sure you found Fish's Undead Rising enchanted books in chests? It's not supposed to be there.

**Comment by Reziax** [comment]
> Regarding the third issue, are you sure you found Fish's Undead Rising enchanted books in chests? It's not supposed to be there.

Yes 100%. They seem to be extremely rare. Just to put it into perspective: We raided so goddamn many dungeons already and in combination with a hilariously sophisticated fish trap farm someone built (which only ran for a little bit but still), the big server is getting to almost a dozen diamond chests full of enchantments across all players (that I know of). This count is after a HUGE amount of them has been turned into eplus hover book decorations... And we only looted 5 fish's enchants in total over all that time... while having an almost full diamond chest only for mending.
I personally found 2 of them in the same structure/dungeon, 1x Corrosive 4 or 5 idr, 1x Lifesteal 2. The other 3 were Lifesteal, idk how or where they were found and we have yet to find the Poisonous one. In case you wondered; they were spread around and traded by use of bibliocraft typesetting table and printing press.
I will report back when I get more clues.

**Comment by xJon** [comment]
@Reziax I managed to find what caused this. It seems that enchantment IDs reshuffle when new mods are added, which incidentally caused these enchanted books to be a part of Hexxit II's loot (instead of other enchanted books). Thank you for reporting it - will be addressed in the next update whenever it comes.

**Comment by Reziax** [comment]
@xJon Funny, I noticed how random the IDs were when playing around with commands and tags. I didn't actually notice enchants missing lol. Also, if you get to read my long ass suggestion post, maybe you could address it the way I mentioned, by actually adding them intentionally maybe as exclusively lootable books or a lower max level if you think it would be too stronk.

**Comment by xJon** [comment]
@Reziax Can you use the heart to repair Tinkers tools? I assume it's intentional, as they can't be repaired in an anvil.

**Comment by Reziax** [comment]
> @Reziax Can you use the heart to repair Tinkers tools? I assume it's intentional, as they can't be repaired in an anvil.
@xJon 

So, it's not exactly intentional, rather a limitation since according to fish's dev, it appears to be difficult to access tinker's durability handling. I personally asked him on his discord, screenshot [here](![grafik](https://github.com/xJon/Hexxit-II/assets/155022491/2b132eab-8294-4ec6-ac31-6581b1d97b03))
Just out of curiosity, do you know a mod that succesfully achieves tinker tools or armor durability control? It's hard to imagine it hasn't been done in all these years. I just thought, maybe if there is code that can do that, it could be reusable for other modders.

Also, updated the post with new findings, see No. 5.

**Comment by xJon** [comment]
Fixing the enchantment IDs right now, finally, thank you again for the report 💯 (also thanks to @friendlyhj & @dshadowwolf over Discord for `game.getEnchantment().id`)

**Comment by xJon** [comment]
Most of these issues are now resolved in update `1.6.0`! I believe that the BiblioCraft and Tinkers issues are still present though. Not much I can do right now, although if they are forked in the future I'll be sure to include the updates with the assumed fixes.
